CONTRIBUTING.md

```diff
@@ -108,9 +108,23 @@ To run the website on your local machine on a development server with live reloa
 - run the released (stable) version of the documentation execute `npm run build-local-stable -y`
 - run the latest (work in progress) version use `npm run build-local-latest -y`
 
-As the compilation can take a few minutes due to the size of the docs, wait for the "VuePress dev server listening at http://localhost:8080 (or another available port)" message.
+As the compilation can take a few minutes due to the size of the docs, wait for the "VuePress dev server listening at <http://localhost:8080> (or another available port)" message.
 When loading the website in the browser it may take a few seconds until it finally appears.
 
+### Build documentation from PR
+
+In special situations; e.g. if you make changes in the documentation repository that might impact the website build, it is possible to build the website from a PR. This makes it possible to verify that your proposed changes do not have negative side effects:
```

Comment:
👍 common mistake of mine; will change this ➡️ unfortunately I did not find a rule yet for the markdown linter for vs code that prevents this in the 1st place.

Reply:
Btw. I used some `html` to make the page more compact, in case there are no problems with the `ruby` installation. Not sure if we allow the use of such elements. Else we might consider a separate page for such special cases to keep the page compact (not to scare potential contributors away because it looks more complicated than it actually is ...)

Reply:
> Not sure if we allow the use of such elements. 

Those elements are allowed and sometimes used in the docs as well :+1: